CVLByExamples/ReadOnlyReentrancy/certora/spec/ReadOnlyReentrancy.spec

```diff
@@ -0,0 +1 @@
+use builtin rule viewReentrancy;
```

Comment:
do we have any documentation on this? I think I saw something, maybe we can point to it

Reply:
not yet, I think we need to work on this next week.